tests/drivers/spi/spi_dma_loopback/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
couldn't the tests/drivers/spi/spi_loopback test be used already for this? ST is testing their DMA support there and it's not an issue.

Reply:
right, I need add an overlay_dma.conf for this, as nxp spi can work with dma and without dma

Reply:
@hakehuang - might makes sense to add "mcux" somewhere to the name of the test, because it seems to be very specific to this platform.

Reply:
@cfriedt this dedicated test has been removed, I merge it to the origin case

Reply:
I believe that also addresses @tbursztyka's concerns, so going to make this resolved.